data/make_parquet.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import argparse
+import os 
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--language", default="en", type=str, help="Language (en/ru)")
+parser.add_argument("--directory", default="", type=str, help="directory where wiki dump is located")
+parser.add_argument("--num_lines", default=500000, type=int, help="Number of lines to keep in dataset")
+
+def main(args: argparse.Namespace) -> None:
+
+
+# Write first args.num_lines to a csv file. 
+    count =0
+    with open('{}.csv'.format(args.language), "w") as file:
```

Comment:
The f-strings are now typically preferred over using `format`.
```suggestion
    with open(f'{args.language}.csv', "w") as file:
```